README.md

```diff
@@ -23,15 +17,25 @@ git clone https://github.com/hackersatcambridge/hac-website.git
 And then run:
 
 ```
-npm install
+docker-compose up
 ```
 
-followed by:
+This will build all the files and run the web server with appropriate output. It will also watch for changes to the files and rebuild as necessarily. You can stop this process at any time with `^C` (`ctrl` + `C`).
+
+### What `docker-compose up` is doing
+
+This command starts up a docker container that has all the required dependencies, then starts up a development server.
+
+Internally, this command runs the Swift and static file (CSS, images, etc.) build processes in parallel. When they complete, it runs the executable produced by the Swift build. It exposes a server on http://localhost:3000. For convenience, it will reload the browser window for you whenever you make a relevant change to the files. This is powered by BrowserSync, and you can access the interface for it at http://localhost:3001.
```

Comment:
Maybe we should mention that it is using Gulp internally. e.g. "Internally, this command uses the Gulp task runner to..." I think the risk with dockerising is that new contributors don't actually know where to start looking when they want to change something because the architecture becomes more opaque. Perhaps worth also pointing people towards the Dockerfile and gulpfile if they want to learn more